src/engine/video/video.h

```diff
@@ -332,6 +322,15 @@ class VideoEngine : public vt_utils::Singleton<VideoEngine>
     void EnableTexture2D();
     void DisableTexture2D();
 
+    //! Enables the secondary render target.
+    void EnableSecondaryRenderTarget();
+
+    //! Disables the secondary render target.
+    void DisableSecondaryRenderTarget();
+
+    //! Draws the secondary render target onto the primary render target.
+    void DrawSecondaryRenderTarget();
```

Comment:
As this is actually disabling the secondary render target, maybe this should be stated, right?